src/main.c

```diff
@@ -104,6 +111,13 @@ int main(int argc, char **argv)
 		snappy_udev_cleanup(&udev_s);
 #endif				// ifdef STRICT_CONFINEMENT
 
+		// Try to re-locate back to vanilla working directory. This can fail
+		// because that directory is no longer present. In that case print a
+		// diagnostic message and carry on without failing.
```

Comment:
I want to fix this because without it spread (as packaged in a snap) is useless.